src/Microsoft.ML.Data/Transforms/Hashing.cs

```diff
@@ -834,6 +1109,99 @@ private void AddMetaKeyValues(int i, DataViewSchema.Annotations.Builder builder)
             }
 
             protected override Delegate MakeGetter(DataViewRow input, int iinfo, Func<int, bool> activeOutput, out Action disposer) => _parent.GetGetterCore(input, iinfo, out disposer);
+
+            private bool SaveAsOnnxCore(OnnxContext ctx, int iinfo, string srcVariable, string dstVariable)
```

Comment:
>SaveAsOnnxCore [](start = 25, length = 14)

The case of key type inputs is still not handled here: if the input is a key type, then 0 is mapped to 0. If we would like to support this case then it would need to involve an "If" node in the ONNX graph. Otherwise, this method should check whether the input type is key and return false in this case.

Reply:
I'll make another PR for key datatypes, as I think it'll be more involved.